agent/src/beerocks/slave/beerocks_slave_main.cpp

```diff
@@ -186,21 +186,25 @@ static void son_slave_watchdog(beerocks::config_file::sConfigSlave &beerocks_sla
                                const std::unordered_map<int, std::string> &interfaces_map)
```

Comment:
Please remove `beerocks::config_file::sConfigSlave &beerocks_slave_conf` as part of this commit if it is unused.
Same for all functions which used to use the conf file and now do not.

Reply:
this specific location uses the `beerocks::config_file::sConfigSlave &beerocks_slave_conf` for the `.temp_path` field.
I will change it to pass only the `temp_path` field instead of the whole struct.

Reply:
other than this location, the rest of the locations still require the `beerocks_slave_conf` as they are using multiple fields from the struct.